compact/optical_materials.xml

```diff
@@ -8,42 +8,42 @@
       5.1*eV 1.0
       "/>
     <matrix name="RINDEX__AirOptical" coldim="2" values="
-      1.90745*eV  1.0
-      1.93725*eV  1.0
-      1.968*eV    1.0
-      1.99974*eV  1.0
-      2.03253*eV  1.0
-      2.0664*eV   1.0
-      2.10143*eV  1.0
-      2.13766*eV  1.0
-      2.17516*eV  1.0
-      2.214*eV    1.0
-      2.25426*eV  1.0
-      2.296*eV    1.0
-      2.33932*eV  1.0
-      2.38431*eV  1.0
-      2.43106*eV  1.0
-      2.47968*eV  1.0
-      2.53029*eV  1.0
-      2.583*eV    1.0
-      2.63796*eV  1.0
-      2.69531*eV  1.0
-      2.7552*eV   1.0
-      2.81782*eV  1.0
-      2.88335*eV  1.0
-      2.952*eV    1.0
-      3.024*eV    1.0
-      3.0996*eV   1.0
-      3.17908*eV  1.0
-      3.26274*eV  1.0
-      3.35092*eV  1.0
-      3.44401*eV  1.0
-      3.54241*eV  1.0
-      3.64659*eV  1.0
-      3.7571*eV   1.0
-      3.87451*eV  1.0
-      3.99949*eV  1.0
-      4.13281*eV  1.0
+      1.90745*eV  1.00029
+      1.93725*eV  1.00029
+      1.968*eV    1.00029
+      1.99974*eV  1.00029
+      2.03253*eV  1.00029
+      2.0664*eV   1.00029
+      2.10143*eV  1.00029
+      2.13766*eV  1.00029
+      2.17516*eV  1.00029
+      2.214*eV    1.00029
+      2.25426*eV  1.00029
+      2.296*eV    1.00029
+      2.33932*eV  1.00029
+      2.38431*eV  1.00029
+      2.43106*eV  1.00029
+      2.47968*eV  1.00029
+      2.53029*eV  1.00029
+      2.583*eV    1.00029
+      2.63796*eV  1.00029
+      2.69531*eV  1.00029
+      2.7552*eV   1.00029
+      2.81782*eV  1.00029
+      2.88335*eV  1.00029
+      2.952*eV    1.00029
+      3.024*eV    1.00029
+      3.0996*eV   1.00029
+      3.17908*eV  1.00029
+      3.26274*eV  1.00029
+      3.35092*eV  1.00029
+      3.44401*eV  1.00029
+      3.54241*eV  1.00029
+      3.64659*eV  1.00029
+      3.7571*eV   1.00029
+      3.87451*eV  1.00029
+      3.99949*eV  1.00029
```

Comment:
My understanding, is that we don't need anything but the endpoints.
```suggestion
```

Reply:
Yes, that's right. I will condense ABSLENGTH too.

Reply:
(PFRICH and DRICH are guilty of doing this too, but we plan to parameterize those soon anyway)

Reply:
but that is not a good reason to not clean them up too... done!